### ISSUE TITLE
Please give us a non heroku install

### ISSUE THREAD

**Issue by gerroon** [siwapp/siwapp-ror#308]
HI

I am just a user and I do not know what heroku is, I try to follow your install guide asking me my heroku creds. I am not a dev and I should not be providing some 3rd part credfs to insall this app at least last time I installed this long time ago I did not need it.

It would be nice if you can provide straight forward installation for non dev users. If that is the only way then I guess I will personally need to move on to another application.

thanks

**Comment by dnwilson** [comment]
Trying to deploy to digitalocean with dokku any tips?

**Comment by jmblasco** [comment]
It seems impossible to install in another linux vps.

**Comment by jmblasco** [comment]
Yes, of course, Dane. Publish it asap. I'll try in my own vps (aruba.it).

**Comment by jvasquezgeain** [comment]
Dear sir, I need to deploy the siwapp project on a centos server. The siwapp (RoR) project does not allow me to perform the deployment on my server using the bundle procedure.

You can provide me with some documentation about the deployment on a server other than heroku.

**Comment by dnwilson** [comment]
@jvasquezgeain @jmblasco this might help. This is taking into consideration that you've setup dokku on your VPS https://github.com/siwapp/siwapp/pull/310

**Comment by dnwilson** [comment]
@proptrading sorry about the delay. You need to add an app.json file.
"predeploy": "bundle exec rake db:setup" - is the key line in this file and is only necessary for the very first deploy. 

```

{
  "name": "siwapp",
  "description": "Online Invoice Management",
  "keywords": [
    "dokku",
    "rails",
    "siwapp"
  ],
  "scripts": {
    "dokku": {
      "predeploy": "bundle exec rake db:setup",
      "postdeploy": "bundle exec rake db:migrate"
    }
  }
}
```

**Comment by Timothy13Munyao** [comment]
anyone tried hosting on google app engine?

**Comment by m3astwood** [comment]
Instructions for a basic install on a barebones linux machine would be great too.

**Comment by sonic182** [comment]
> Instructions for a basic install on a barebones linux machine would be great too.

The default instructions for ruby on rails will work

The only extra stuff is to do `rake db:setup` for the first setup of the app
